modules/ossm-auto-route-enable.adoc

```diff
@@ -7,6 +7,16 @@
 
 By default, the `ServiceMeshControlPlane` resource automatically synchronizes the Gateway resources with OpenShift routes. Disabling the automatic route creation allows you more flexibility to control routes if you have a special case or prefer to control routes manually.
 
+[id="disabling-automatic-route-creation-specific-cases"]
```

Comment:
[id="disabling-automatic-route-creation-specific-cases_{context}"]

Needs to have context at the end of each ID, not just the first one in the file, as shown in the guidelines.
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Reply:
@JStickler this has been updated. I looked at anchoring in an assembly by mistake. Thanks!